mir-module/src/main/resources/xsl/editor/xeditor2mods.xsl

```diff
@@ -215,7 +215,7 @@
           <xsl:copy-of select="node()[name()!='mods:namePart']" />
```

Comment:
I would have switched to `apply-templates` here too. 

Since `copynodes.xsl` is included, the default behavior is to copy anyway. Switching to `apply-templates` allows modifications in the future. I see no reason, not to allow this in both cases.

This inclusion `copynodes.xsl` predates this template. I sincerely believe that the use of `copy-of` in this template was just an oversight. An oversight that should be corrected. The issue that originally lead to to this PR is just a symptom of this root cause.